hr_attendance_report/security/ir.model.access.csv

```diff
@@ -0,0 +1,5 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+hr_attendance_report.access_employee_attendance_report_wizard,access_employee_attendance_report_wizard,hr_attendance_report.model_employee_attendance_report_wizard,hr.group_hr_user,1,0,0,0
```

Comment:
If you want employees to print the report, they will need write access to the wizard.